src/routes/settings/assistants/[assistantId]/ReportModal.svelte

```diff
@@ -0,0 +1,57 @@
+<script lang="ts">
+	import { applyAction, enhance } from "$app/forms";
+	import { invalidateAll } from "$app/navigation";
+	import Modal from "$lib/components/Modal.svelte";
+	import { createEventDispatcher } from "svelte";
+
+	const dispatch = createEventDispatcher<{ close: void }>();
+
+	let reason = "";
+</script>
+
+<Modal on:close>
+	<form
+		method="POST"
+		action="?/report"
+		use:enhance={() => {
+			return async ({ result }) => {
+				await applyAction(result);
+				dispatch("close");
+				invalidateAll();
+			};
+		}}
+		class="w-full min-w-64 p-4"
+	>
+		<span class="mb-1 text-sm font-semibold">Report an assistant</span>
+
+		<p class="text-sm text-gray-500">
+			Please provide a brief description of why you are reporting this assistant.
+		</p>
+
+		<input
```

Comment:
maybe `textarea` would be a better choice since with `input`, you can't write multi-line texts. In this case, `textarea` should have max-height

Reply:
I'd like to encourage people to keep it short, since it's 128 chars max but maybe a small textarea is still better indeed

Reply:
done!